### ISSUE TITLE
Bug on Android with conditional rendering Picker.Item

### ISSUE THREAD

**Issue by olvipi** [react-native-picker/picker#175]
Hi guys. I have a problem on Android, when I use conditional rendering for Picker.Item, then the selectedValue index is incorrect because the missed React.Children elements are also counted.
Code example:

```jsx
function BasicPickerExample() {
  const [value, setValue] = React.useState('key1')
  return (
    <Picker
      testID="basic-picker"
      selectedValue={value}
      onValueChange={(v) => setValue(v)}
   >
      {false && <Item label="key0" value="key0" />}
      <Item label="key1" value="key1" />
      <Item label="key2" value="key2" />
    </Picker>
  )
}
```

Here's the PR that fixes it #174

**Comment by olvipi** [comment]
#174

**Comment by Naturalclar** [comment]
Fixed in #174 thanks for the PR!
